src/predict_request_validation_utils.cpp

```diff
@@ -520,17 +529,15 @@ static size_t getStringBatchSize(const InferenceTensor& src) {
 
 // To be called only for already validated proto against models with two dimensions.
 template <typename RequestType, typename InputTensorType, typename IteratorType, typename ShapeType>
-Status RequestValidator<RequestType, InputTensorType, IteratorType, ShapeType>::checkStringShapeMismatch(const InputTensorType& proto, const ovms::TensorInfo& inputInfo, Status& finalStatus, Mode batchingMode, Mode shapeMode) const {
+Status RequestValidator<RequestType, InputTensorType, IteratorType, ShapeType>::checkStringShapeMismatch(const InputTensorType& proto, const ovms::TensorInfo& inputInfo, Status& finalStatus, Mode batchingMode, Mode shapeMode, size_t inputBatchSize, size_t inputWidth) const {
     const auto& shape = inputInfo.getShape();
     bool mismatch = false;
-    size_t width = getStringInputWidth(proto);
-    size_t batchSize = getStringBatchSize(proto);
```

Comment:
In line 519 and 523 we have narrowed implicit cast which can cause errors when provided negative value.
src.contents().bytes_contents_size() / src.string_val_size() return signed int but we cast to size_t